test/testdata/resolver/mixes_in_class_methods_twice.rb

```diff
@@ -7,19 +7,12 @@
 # from the second, incrementalResolve, pass).
 #
 # If the second one we see is the same as the one we saw previously, this is
-# actually fine. But if the class is different, that's an error.
+# actually fine.
 
 module A
+  module ClassMethods; end
   extend T::Helpers
-  module ClassMethods1; end
-  module ClassMethods2; end
-  mixes_in_class_methods(ClassMethods1)
-  mixes_in_class_methods(ClassMethods2) # error: Redeclaring `mixes_in_class_methods` from module `A::ClassMethods1` to module `A::ClassMethods2`
-end
 
-module B
-  extend T::Helpers
-  module ClassMethods; end
```

Comment:
Instead of deleting these tests, could you re-enable them and remove the error comments instead?